include/cairo_renderer.h

```diff
@@ -2,7 +2,7 @@
 #define _SWAY_CAIRO_H
```

Comment:
The include guard needs to be updated as well.

Maybe `cairo_util.h` would be a better name?

Reply:
I chose `cairo_renderer.h` just like @Cloudef did in https://github.com/Cloudef/bemenu/commit/ebd7338bd5653a56d319a3a879f97e867413e58b. Should I rename it to `cairo_util.h`?

Reply:
Yes. "Renderer" doesn't make much sense to me for Sway.